### PR TITLE
example cron job script

### PR DESCRIPTION
**Pull request by telnetdoogie**
I used this for adding cron jobs on startup

### CODE REVIEW

**Comment by telnetdoogie** [comment]
This provides a _reasonable_ solution for #176

**Comment by pgorod** [comment]
Thanks, this is useful.

It will probably also be helpful with another situation I am facing: the UDM UI becomes sluggish and then inaccessible after a few days operational. A UDM reboot is required frequently to get things back in place. I thought about adding a cronjob to reboot every X days in the middle of the night.

The only reason I didn't go ahead with this plan is because I think it totally sucks and I really wish I could get to the bottom of this, find out the cause of the problem and eliminate it altogether. I don't want to have periodic reboots! 🤮 I still want to investigate this better. But if I ever need them, this cronjob method should do the trick. 👍

**Comment by telnetdoogie** [comment]
@pgorod yeah you could probably get some benefit from just doing a unifi-os restart versus a whole-machine reboot. That'll take less time and will be a bit less intrusive.

File: on-boot-script/examples/udm-files/on_boot.d/25-add-cron-jobs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# store crontab files in /mnt/data/cronjobs/ (you will need to create this foler)
+# this script will re-add them on startup
+
+cp /mnt/data/cronjobs/* /etc/cron.d/
+/etc/init.d/crond restart
```
Comment (pgorod) [inline on-boot-script/examples/udm-files/on_boot.d/25-add-cron-jobs:6]: Perhaps a `reload` would be better than a full `restart`. But I haven't tested it.

**Review by pgorod — APPROVED** [review]

File: on-boot-script/examples/udm-files/on_boot.d/25-add-cron-jobs (hunk 1 above)
Comment (telnetdoogie) [reply on-boot-script/examples/udm-files/on_boot.d/25-add-cron-jobs:6]: `crond reload` for this distro requires the reloading of a specific file to reload, like
`/etc/init.d/crond reload [cron_file]`
 which would make for a much more complex script.
the crond restart isn't particularly taxing:
`# time /etc/init.d/crond restart
Stopping cron ...done.
Starting cron ... done.
real    0m 1.01s
user    0m 0.01s
sys     0m 0.00s
`

**Pull request merged**